sdk/jobs/automl-standalone-jobs/automl-image-classification-multilabel-task-fridge-items/automl-image-classification-multilabel-task-fridge-items.ipynb

```diff
@@ -373,12 +373,96 @@
+    "\n",
+    "Automatic sweeps are triggered by setting `max_trials` to a value greater than 1 in limits and by omitting the hyperparameter space, sampling method and termination policy."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# general job parameters\n",
+    "exp_name = \"dpv2-image-classif-multi-label-experiment\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML job with the related factory-function.\n",
+    "\n",
+    "image_classification_multilabel_job = automl.image_classification_multilabel(\n",
+    "    compute=compute_name,\n",
+    "    # name=\"dpv2-image-classif-multi-label-job-02\",\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    target_column_name=\"label\",\n",
+    "    primary_metric=\"iou\",\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "image_classification_multilabel_job.set_limits(\n",
+    "    timeout_minutes=60, max_trials=10, max_concurrent_trials=2\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Submitting an AutoML job for Computer Vision tasks\n",
+    "Once you've configured your job, you can submit it as a job in the workspace in order to train a vision model using your training dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Submit the AutoML job\n",
+    "returned_job = ml_client.jobs.create_or_update(image_classification_multilabel_job)\n",
+    "\n",
+    "print(f\"Created job: {returned_job}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.stream(returned_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4.2. Individual runs\n",
+    "\n",
+    "If automatic sweeps do not meet your needs, you can launch individual runs to explore a few model algorithms; we provide sensible default hyperparameters for each algorithm. You can also launch individual runs for the same model algorithm and a few hyperparameters.\n",
     "\n",
     "Following functions are used to configure the AutoML image job:\n",
     "\n",
```

Comment:
same comment for moving the description of set_limits, etc APIs that are used in automode before automode code cell...
(* for all other notebooks as well...)

Reply:
Done.